amazon-redshift/1.0/modules/ROOT/pages/amazon-redshift-connector-studio.adoc

```diff
@@ -0,0 +1,135 @@
+= Use Anypoint Studio to Configure Amazon Redshift Connector 1.0 - Mule 4
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+. <<add-source,Add a source>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<view-app-log,View the app log>>.
+
+When you run the connector, you can view the app log to check for problems, as described in <<view-app-log,View the App Log>>.
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Anypoint Connector for Amazon Redshift (Amazon Redshift Connector) to your Mule project to automatically populate the XML code with the connector's namespace and schema location and add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type `redshift` in the search field.
+. Click *Amazon Redshift Connector* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, it's best to configure a global element that all instances of that connector in the app can use.
+Configuring a global element requires you to provide the authentication credentials that the connector requires to access the target Amazon Redshift cluster.
+
+Amazon Redshift Connector supports:
+
+* Basic authentication, which uses *user* and *password* field values to connect to the database
+* IAM authentication, which uses *access key* and *secret key* field values to retrieve temporary password for the specified user in order to connect to the database
+** Optionally, you can specify *ARN* field value in the *Role* field if the connector should first assume an IAM role for a set of temporary security IAM credentials and use those to retrieve the password.
```

Comment:
Formatting is funky - this should be a list continue and not a second-level bullet. 

"you can specify *ARN* field value" > "you can specify the *ARN* field value"